docs/topics/cache.txt

```diff
@@ -363,6 +363,16 @@ user ``apache``, make sure the directory ``/var/tmp/django_cache`` exists and
 is readable and writable by the user ``apache``, or that it can be created by
 the user ``apache``.
 
+.. warning:: 
+
+    The filesystem cache is intended as an easy way to test caching, not as a 
+    serious caching strategy. It is not designed to handle a large number of 
```

Comment:
IMO, this wording is too strong, see also [comment](https://code.djangoproject.com/ticket/11260#comment:9).

Reply:
I'll rephrase :)